whiskey.yml

```diff
@@ -11,7 +11,7 @@ Build:
 - Version:
     Path: Whiskey\Whiskey.psd1
     Prerelease:
-    - develop: beta$(WHISKEY_BUILD_NUMBER)
+    - release: beta$(WHISKEY_BUILD_NUMBER)

```

Comment:
While I know this is following the Gitflow model, it's confusing at first since the `release` branch sounds like where you ship final releases.

Reply:
What should we use for the branch name? `beta`?

Reply:
`beta` works or even the aptly name `prerelease` ?

Reply:
Changed to `prerelease`.